viaforge-mc1122/src/main/resources/mcmod.info

```diff
@@ -4,15 +4,12 @@
     "name": "ViaForge",
     "description": "Client-side Implementation of the Via* projects for Minecraft Forge",
     "version": "${version}",
-    "mcversion": "[1.12.2]",
+    "mcversion": "[1.12,)",
     "url": "https://github.com/ViaVersion/ViaForge",
-    "updateUrl": "",
```

Comment:
maybe we can fill this with some correct value?

Reply:
just delete it, it's almost useless for end users